db/structure.sql

```diff
@@ -8,20 +8,6 @@ SET check_function_bodies = false;
 SET client_min_messages = warning;
 SET row_security = off;
 
---
```

Comment:
sigh. our dbs do not agree.

Reply:
@hackmastera I will put your stuff back.  The comments do not cause me any issue.

Reply:
👍